src/app/components/FilterPopup/FilterPopup.jsx

```diff
@@ -40,6 +42,7 @@ class FilterPopup extends React.Component {
       filters: [],
       showForm: false,
       js: false,
+      selectedFacets: Store.getState().selectedFacets,
```

Comment:
Check my other PR to see how to do this.

Reply:
What are the selected facets called now? I made this here because when I was working on this PR, I didn't see selected facets were passed to this component. If now we passed them from a higher level, of course we don't need to call the store here.

Reply:
Review the other PRs.